xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1210 @@
+     <para>
+      Transfer the container image data to the internal mirror. This will remove
+      all files and directories that are no longer present on the trusted storage
+      from the internal mirror. If you have added any images to the registry
+      manually, please back up these first and restore after the sync from
+      the trusted storage is done.
+     </para>
+     <para>
+      The target directory must be the same directory you have configured during
+      the setup of the container registry. Here <filename>/tmp/registry</filename>.
+     </para>
+<screen>&prompt.user;<command>rsync -aP <replaceable>/mnt/storage/container-images/</replaceable> <replaceable>/tmp/registry/</replaceable>  --delete</command>
+     </screen>
+    </step>
+    <step>
+     <para>
+      Transfer the helm chart data to the internal mirror. This will remove
+      all charts that do not exist on the trusted storage. If you have added
+      any charts to the location manually, please back up these first and
+      restore after the sync from the trusted storage is done.
+     </para>
+<screen>&prompt.user;<command>rsync -aP <replaceable>/mnt/storage/charts</replaceable> <replaceable>/tmp/charts/</replaceable> --delete</command>
+     </screen>
+    </step>
+   </procedure>
+
+   <procedure>
+    <title>Refresh information on the &productname; cluster</title>
+    <step>
+     <para>
+      Update the repository information on the machine on which you are using
+      Helm to install software to the cluster.
+     </para>
+<screen>&prompt.user;<command>helm repo update</command>
+      </screen>
+    </step>
+   </procedure>
+
+   <para>
+    You can now deploy additional software on your &productname;. Refer to:
+    <xref linkend="sec.admin.software.install" />.
+   </para>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.caasp-deployment">
+   <title>Deploying &productname;</title>
+   <para>
+    Use the &productname; <xref linkend="book.caasp.deployment"/> as usual.
+    Some of the considerations below apply; depending of the chosen installation
+    medium.
+    </para>
+   <sect3>
+    <title>Using the ISO</title>
+    <para>
+     From YaST register the node against the RMT server. This will ensure the
+     final node zypper repositories are pointed against RMT, moreover all the
+     available updates are going to be installed (-> no need to run a
+     transactional-update right after the installation -> way faster).
+    </para>
+   </sect3>
+   <sect3>
+    <title>Using AutoYast</title>
+    <para>
+     Ensure the admin node is registered against RMT, that will ensure the
+     nodes that are provisioned by AutoYaST are registered against RMT to have all
+     the updates applied
+    </para>
+   </sect3>
+   <sect3>
+    <title>Using a prebuilt image (eg: KVM, Xen)</title>
+    <para>
+     The node has to be registered against RMT. This should be done in the same
+     way as a regular SLE machine: via SUSEConnect.
```

Comment:
Stupid questions:
  * don't we have a macro for `SLE`?
  * `SUSEConnect` (command markup)